sdk/messaging/azservicebus/example_receiver_test.go

```diff
@@ -86,3 +86,29 @@ func ExampleReceiver_ReceiveMessages() {
 		fmt.Printf("Received and completed the message\n")
 	}
 }
+
+func ExampleReceiver_DeadLetterMessage() {
+	// DeadLetterMessage settles a message by moving it to the dead letter queue.
+	deadLetterReason := "exampleReason"
+	deadLetterErrorDescription := "exampleErrorDescription"
+
+	deadLetterOptions := &azservicebus.DeadLetterOptions{
+		Reason:           &deadLetterReason,
+		ErrorDescription: &deadLetterErrorDescription,
+	}
+
+	ctx, cancel := context.WithTimeout(context.TODO(), 60*time.Second)
+	defer cancel()
+
+	message, err := receiver.ReceiveMessages(ctx, 1, nil)
```

Comment:
```suggestion
	messages, err := receiver.ReceiveMessages(ctx, 1, nil)
```
It's still a slice of messages, even if you just request one.